twake/frontend/src/app/features/users/types/user.ts

```diff
@@ -41,9 +49,9 @@ export type UserType = {
   timezone_offset?: string;
   tutorial_status?: any;
   username: string;
-  companies?: { company: CompanyType; role: CompanyRoleType; status: CompanyStatusType }[];
+  companies?: UserCompanyType[];
   preferences: UserPreferencesType;
-  workspaces?: WorkspaceType[];
+  workspaces?: UserWorkspaceType[];
```

Comment:
Here we should add some comments to precise that this fields are filled when available and so we cannot rely on it except on search service for filtering